app/models/product_accessory.rb

```diff
@@ -1,6 +1,10 @@
 class ProductAccessory < ActiveRecord::Base
 
-  SCALING_TYPES = %w(quantity manual auto).freeze
+  SCALING_TYPES = {
+    item: ["quantity"],
+    service: ["quantity"],
+    timed_service: ["manual", "auto"]
+  }.with_indifferent_access
```

Comment:
Part of me feels like the scaling types should be the keys and the values the product types. It might also be nice to have that hash be able to default to "quantity". That way if we add a new product type, we don't have to worry about accessories unless it's timed.